Patch to build _dotblas.c when ATLAS is not installed.

Added by: Tiziano Zito <otizonaizit@yahoo.com>
Index: python-numpy-1.1.0.new/numpy/core/setup.py
===================================================================
--- python-numpy-1.1.0.old/numpy/core/setup.py 2008-05-26 09:39:44.000000000 +0200
+++ python-numpy-1.1.0.new/numpy/core/setup.py 2008-07-08 13:37:23.000000000 +0200
@@ -291,8 +291,8 @@
     #blas_info = {}
     def get_dotblas_sources(ext, build_dir):
         if blas_info:
-            if ('NO_ATLAS_INFO',1) in blas_info.get('define_macros',[]):
-                return None # dotblas needs ATLAS, Fortran compiled blas will not be sufficient.
+            #if ('NO_ATLAS_INFO',1) in blas_info.get('define_macros',[]):
+            #    return None # dotblas needs ATLAS, Fortran compiled blas will not be sufficient.
             return ext.depends[:1]
         return None # no extension module will be built
 
